harness/features/allFeatures.local.test.ts

```diff
@@ -0,0 +1,113 @@
+import { 
+    getConnectionStringForProxy, 
+    forEachSDK, 
+    describeIf, 
+    createUser,
+    TestClient,
+    wait
+} from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../nock'
+import { config, expectedFeaturesVariationOn } from '../mockData/config'
+
+jest.setTimeout(10000)
+
+const scope = getServerScope()
+
+describe('allFeatures Tests - Local', () => {
+    forEachSDK((name) => {
+        let url: string
+        const capabilities: string[] = SDKCapabilities[name]
+        
+        let variationOnUser: string
+        let noVariationUser: string
+        let invalidUser: string
+
+        beforeAll(async () => {
+            url = getConnectionStringForProxy(name)  
+            
+            variationOnUser = (
+                await createUser(url, { user_id: 'user1', customData: { 'should-bucket': true } })
+            ).headers.get('location')
+
+            noVariationUser = (
+                await createUser(url, { user_id: 'user3' })
+            ).headers.get('location')
+
+            invalidUser = (
+                await createUser(url, { name: 'invalid' })
+            ).headers.get('location')
+        })
+
+        describeIf(capabilities.includes(Capabilities.local))(name, () => {
+            describe('uninitialized client', () => {
+                const testClient = new TestClient(name)
+
+                beforeAll(async () => {
+                    await testClient.createClient()
+                    const configRequestUrl = `/${testClient.clientLocation}/config/v1/server/${testClient.sdkKey}.json`
+                    const interceptor = scope
+                        .get(configRequestUrl)
+    
+                    interceptor.delay(2000).reply(404)
```

Comment:
dont think you need the delay here either (like in the other PR)

Reply:
oh oops i removed it but didn't commit it omg - fixed